swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -1036,12 +1038,16 @@ private void updateRowset() {
 	 * currently installed on the given <code>comboBox</code>. This is the only
 	 * technique we can rely on to prevent the <code>comboBox</code> from
 	 * broadcasting {@link ActionEvent}s at inappropriate times.
-	 *
+	 * <p>
 	 * This method is the logical inverse of {@link #registerAllActionListeners}.
+	 * <p>
+	 * Note that the ActionListener used for binding will be removed without
+	 * passing through SSCommon.removeSSComponentListener()
 	 * 
 	 * @param comboBox combo box from which to remove listeners
 	 * @return array of ActionListeners removed from combo box (for adding back later)
 	 */
+	// TODO: Consider passing in SSBaseComboBox and identifying SSBaseComboBoxListener
```

Comment:
Since register/unregisterAllActionListeners() is a temporary thing, I don't think there is any impact on the binding so decided to leave them alone.

Reply:
> TODO: Consider passing in SSBaseComboBox and identifying SSBaseComboBoxListener

I don't understand what this comment, for both methods, is about. What's the goal? (and it's been a long time)

Reply:
I was contemplating isolating/identifying and removing the component listener used for rowset binding separately and pushing those calls through SSCommon for logging/tracking. Since this is just done momentarily and  then added back, I don't think it's worth the effort involved.